connector_importer_product/components/product_product/record_handler.py

```diff
@@ -203,7 +203,6 @@ def _find_attr(self, attr_column, orig_values):
     def _find_attr_value(self, attr, attr_column, orig_values):
```

Comment:
there's a lot of codecov warnings in this file.

Reply:
eh la madonna
![image](https://github.com/OCA/connector-interfaces/assets/13200247/e931b9b1-a935-41a8-9633-ec8bc411b1d1)

Reply:
is not due to this change. I'm adding test coverage in fact.